base/initdefs.jl

```diff
@@ -41,16 +41,63 @@ isinteractive() = (is_interactive::Bool)
 
 ## package depots (registries, packages, environments) ##
 
+"""
+    DEPOT_PATH
+
+A stack of "depot" locations where the package manager, as well as Julia's code
+loading mechanisms, look for package registries, installed packages, named
+environments, repo clones, cached compiled package images, and configuration
+files. By default it includes:
+
+1. `~/.julia` where `~` is the user home as appropriate on the system;
+2. an architecture-specific shared system directory, e.g. `/usr/local/share/julia`;
+3. an architecture-independent shared system directory, e.g. `/usr/share/julia`.
+
+So `DEPOT_PATH` might be:
+```
```

Comment:
```suggestion
```julia
```

Reply:
Don't we assume Julia code by default in our own docstrings?

Reply:
No.

Reply:
Why not?

Reply:
If you don't specify the the language, we assume it's plain text. We _could_ default to `julia`, but then we'd need to introduce `text` or something to remove the highlighting when you don't want it and that feels a bit of a hack.

I also think it is good practice to be explicit about the language -- it makes it easy to copy paste the source between different Markdown processors, the `.md` files on GitHub render correctly etc.